TelCo.ColorCoder/ColorMap.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Diagnostics;
```

Comment:
required? check name spaces to see the necessary ones and remove other. 
The latest of VS compilers does that already for you, its still a good practice.